src/CellularConnectionHandler.cpp

```diff
@@ -51,9 +51,40 @@ UDP & CellularConnectionHandler::getUDP()
    PROTECTED MEMBER FUNCTIONS
  ******************************************************************************/
 
+#if defined(ARDUINO_OPTA) && defined(BOARD_HAS_CELLULAR)
```

Comment:
the following part is not required, since it is present at line 18`&& defined(BOARD_HAS_CELLULAR)`

```suggestion
#if defined(ARDUINO_OPTA)
```